db/migrations/postgres/000002_create_team_members.up.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE IF NOT EXISTS teammembers (
     teamid VARCHAR(26) NOT NULL,
     userid VARCHAR(26) NOT NULL,
-    roles VARCHAR(64),
+    roles VARCHAR(256),
```

Comment:
This is me being lazy to make migration checks pass. Let me know if you want me to provide new migration files.

Reply:
Would prefer that :P

Reply:
@isacikgoz Would you prefer that as well? I don't want to cause more conflicts on your part. Also this should really go on top of everything we haven't updated yet so not sure what ID it will eventually get. Anyway, let me know.

Reply:
Don't want to cause unnecessary pains though. Happy to keep it as is if this will cause more conflicts.

Reply:
I'm now bullet proof to conflicts if your concern is only about the conflicts :)

Reply:
@isacikgoz Done :)